cartoframes/data/observatory/catalog/catalog.py

```diff
@@ -58,86 +58,54 @@ class Catalog(object):
     You can just list all the grouping entities. Take into account this is not the preferred way
     to discover the catalog metadata, since there can be thousands of entities on it:
 
-        .. code::
-
-            from cartoframes.data.observatory import Category
-            from cartoframes.data.observatory import Country
-            from cartoframes.data.observatory import Provider
-
-            Category.get_all()
-            Country.get_all()
-            Provider.get_all()
+    >>> Category.get_all()
+    >>> Country.get_all()
+    >>> Provider.get_all()
 
     Or you can get them by ID:
 
-        .. code::
-
-            from cartoframes.data.observatory import Category
-            from cartoframes.data.observatory import Country
-            from cartoframes.data.observatory import Provider
-
-            Category.get('demographics')
-            Country.get('usa')
-            Provider.get('mrli')
+    >>> Category.get('demographics')
+    >>> Country.get('usa')
+    >>> Provider.get('mrli')
 
     Examples:
         The preferred way of discover the available datasets in the Catalog is through nested filters
 
-        .. code::
-
-            from cartoframes.data.observatory import Catalog
-
-            catalog = Catalog()
-            catalog.country('usa').category('demographics').datasets
+        >>> catalog = Catalog()
+        >>> catalog.country('usa').category('demographics').datasets
```

Comment:
How about giving a preview of what this operation looks like? For example, it would do something like this:
```
[Dataset('blahblah'),
 Dataset('blabhablahhb'),
 ...]
```